tests/conftest.py

```
@@ -1389,6 +1389,55 @@ def factory(
+        patch_data = json.dumps(
+            {"metadata": {"ownerReferences": filtered_refs or None}}
+        )
+        try:
+            instance.ocp.patch(
+                resource_name=instance.name,
+                params=patch_data,
+                format_type="merge",
+            )
+            log.info(
+                f"Stripped StorageClient ownerReferences from "
+                f"StorageClass {instance.name}"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)